game-core/src/main/java/org/triplea/game/server/HeadlessGameServer.java

```diff
@@ -607,79 +586,41 @@ private static void usage() {
+    if (!ClientSetting.mapFolderOverride.isSet()) {
+      ClientSetting.mapFolderOverride.setValue(
+          Paths.get(SystemProperties.getUserHome(), "/triplea/downloadedMaps"));
     }
 
     final String playerName = System.getProperty(TRIPLEA_NAME, "");
-    final String hostName = System.getProperty(LOBBY_GAME_HOSTED_BY, "");
-    if (playerName.length() < 7 || hostName.length() < 7 || !hostName.equals(playerName)
-        || !playerName.startsWith("Bot") || !hostName.startsWith("Bot")) {
-      log.warning(
-          "Invalid argument: " + TRIPLEA_NAME + " and " + LOBBY_GAME_HOSTED_BY
-              + " must start with \"Bot\" and be at least 7 characters long and be the same.");
+    if ((playerName.length() < 7) || !playerName.startsWith("Bot")) {
+      log.warning("Invalid argument: " + TRIPLEA_NAME + " must at least 7 characters long");
       printUsage = true;
     }
 
-    final String comments = System.getProperty(LOBBY_GAME_COMMENTS, "");
-    if (!comments.contains("automated_host")) {
-      log.warning(
-          "Invalid argument: " + LOBBY_GAME_COMMENTS + " must contain the string \"automated_host\".");
+    if (Integer.valueOf(System.getProperty(TRIPLEA_PORT, "0")) <= 0) {
+      log.warning("Invalid argument: " + TRIPLEA_PORT + " must be numeric");
       printUsage = true;
     }
 
-    final String email = System.getProperty(LOBBY_GAME_SUPPORT_EMAIL, "");
-    if (email.length() < 3 || !Util.isMailValid(email)) {
-      log.warning(
-          "Invalid argument: " + LOBBY_GAME_SUPPORT_EMAIL + " must contain a valid email address.");
+    if (System.getProperty(LOBBY_HOST, "").isEmpty()) {
+      log.warning("Invalid argument: " + LOBBY_HOST + " must be set");
       printUsage = true;
     }
 
-    final String reconnection = System.getProperty(LOBBY_GAME_RECONNECTION,
-        "" + LOBBY_RECONNECTION_REFRESH_SECONDS_DEFAULT);
-    try {
-      final int reconnect = Integer.parseInt(reconnection);
-      if (reconnect < LOBBY_RECONNECTION_REFRESH_SECONDS_MINIMUM) {
-        log.warning("Invalid argument: " + LOBBY_GAME_RECONNECTION
-            + " must be an integer equal to or greater than " + LOBBY_RECONNECTION_REFRESH_SECONDS_MINIMUM
-            + " seconds, and should normally be either " + LOBBY_RECONNECTION_REFRESH_SECONDS_DEFAULT
-            + " or " + (2 * LOBBY_RECONNECTION_REFRESH_SECONDS_DEFAULT) + " seconds.");
-        printUsage = true;
-      }
-    } catch (final NumberFormatException e) {
-      log.warning("Invalid argument: " + LOBBY_GAME_RECONNECTION
-          + " must be an integer equal to or greater than " + LOBBY_RECONNECTION_REFRESH_SECONDS_MINIMUM
-          + " seconds, and should normally be either " + LOBBY_RECONNECTION_REFRESH_SECONDS_DEFAULT + " or "
-          + (2 * LOBBY_RECONNECTION_REFRESH_SECONDS_DEFAULT) + " seconds.");
+    if (Integer.valueOf(System.getProperty(LOBBY_PORT, "0")) <= 0) {
+      log.warning("Invalid argument: " + LOBBY_PORT + " must be numeric");
```

Comment:
```suggestion
      log.warning("Invalid argument: " + LOBBY_PORT + " must be greater than zero");
```

If you want to check for non-numeric values, you're going to have to catch `NumberFormatException`.

Reply:
Bumping